keyboards/adkb96/keymaps/default/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define BASE 0
+/*
+#define META 1
+#define SYMB 2
+#define GAME 3
+*/
+// Fillers to make layering more clear
```

Comment:
QMK defines these keycode aliases automatically. Lines 11-13 can be deleted.

https://github.com/qmk/qmk_firmware/blob/d48a5b29d764ee6ac925daf2500bb6101e8b23ca/quantum/quantum_keycodes.h#L29-L31

Reply:
I deleted it.